docs/modules/layers.rst

```diff
@@ -18,6 +18,7 @@
     layers/merge
     layers/embedding
     layers/special
+    layers/autoenc
```

Comment:
I'd prefer to have *special* the last one in the list before the ones that need to be imported explicitly. *special* is the bin for everything falling through the previous categories. So please add *autoenc* before that!